cmd/create/operatorroles/by_clusterkey.go

```diff
@@ -94,13 +93,23 @@ func handleOperatorRoleCreationByClusterKey(r *rosa.Runtime, env string,
 	if err != nil {
 		r.Reporter.Errorf("%s", err)
 	}
+
+	var hostedCPPolicies bool
+	if cluster.Hypershift().Enabled() {
+		hostedCPPolicies, err = r.AWSClient.HasHostedCPPolicies(cluster.AWS().STS().RoleARN())
```

Comment:
If we are taking it from this bool we can probably remove the --hosted-cp param from create operator roles

Reply:
If I'm not wrong we need this param for the flow when we create the operator roles before the cluster creation.

Reply:
But if the client is using hypershift account roles then we are sure to create hypershift operator roles, right?

I think we can just take whatever is from the installer role

Reply:
We are currently using the param to fetch the operator roles for Hypershift https://github.com/openshift/rosa/blob/master/cmd/create/operatorroles/by_prefix.go#L118.

Since the Hypershift roles with managed policies are a hidden flow now, we still need this param, existing Hypershfit account roles don't have the Hypershift tag.

We can change it once we have the managed policies in place, and if we enforce the user to use Hypershift policies for Hypershift clusters.